string name
float64[7] pose
float64[6] velocity
float32 confidence